label_studio_ml/examples/yolo/README.md

```diff
@@ -97,11 +98,13 @@ This tutorial uses the [YOLO example](https://github.com/HumanSignal/label-studi
 
 **Control tags**
 
+- `<Choices>` - [Classification](https://labelstud.io/tags/choices); image classification task
```

Comment:
what order are these in? Does it matter? Should they be the same as the table above? @caitlinwheeless

Reply:
They're following the same order as the sections on the rest of the page. I think the table is sorted by Import Supported.